packages/core/src/Stores/client-store.js

```diff
@@ -745,6 +745,10 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.includes('dxtrade_password_not_set');
     }
 
+    get is_authenticated_with_idv_photoid() {
```

Comment:
Should we add this to 'computed'?

Reply:
yes, I am filtering out a particular account status and returning it. So when account status object changes, this value is re-computed